uwuify/cli.py

```diff
@@ -20,12 +20,12 @@
 
 import click
 
-from uwuify import SMILEY, YU, uwu
+from __init__ import SMILEY, YU, uwu, STUTTER
```

Comment:
this should still be `uwuify`

everything else looks fine though

Reply:
oh, thanks for pointing that out!